vm/x86/tdcall/src/lib.rs

```diff
@@ -455,11 +458,18 @@ pub fn accept_pages<T: Tdcall>(
                         MemoryRange::new(range.start() + x86defs::X64_LARGE_PAGE_SIZE..range.end());
                     continue;
                 }
-                Err(TdCallResultCode::PAGE_SIZE_MISMATCH) => {
-                    #[cfg(feature = "tracing")]
-                    tracing::trace!("accept pages size mismatch returned");
-                }
-                Err(e) => return Err(AcceptPagesError::Unknown(e)),
+                Err(e) => match e {
+                    TdCallResultCode::OPERAND_BUSY => return Err(AcceptPagesError::Busy(e)),
+                    TdCallResultCode::OPERAND_INVALID => return Err(AcceptPagesError::Invalid(e)),
+                    TdCallResultCode::PAGE_ALREADY_ACCEPTED => {
+                        assert_eq!(e, TdCallResultCode::SUCCESS, "page already accepted");
```

Comment:
instead of doing an assertion that will fail, you'd usually use `panic!("page already accepted")` here. I'd also probably add some information to the panic string, such as which page it was that we already accepted, so something like:
`panic!("page {} already accepted", range.start_4k_gpn())`